.github/workflows/pr.yml

```diff
@@ -75,19 +75,19 @@ jobs:
       - uses: bufbuild/buf-setup-action@v1.28.1
       - uses: actions/setup-node@v4
         with:
-          node-version: "18"
+          node-version: "20"
       - run: ./ci/release/dry_run.sh
   python-style:
     name: Style-check and lint Python files
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
-    - name: Install dependencies
-      run: python3 -m pip install --upgrade pip black==22.3.0 flake8==4.0.1
-    - name: Black
-      run: python3 -m black --diff --check .
-    - name: Flake8
-      run: python3 -m flake8 .
+      - uses: actions/checkout@v4
```

Comment:
Does this extra indent do anything?

Reply:
:grimacing: break everything?  It was not intended.  I'll remove.

Reply:
Actually, it appears it was inserted by my yaml formatter.  Indenting lists is consistent with the rest of the file.  Do you mind if we leave it in?

Reply:
Yep, if it's consistent that's good.